core/src/main/java/com/linecorp/armeria/server/DefaultRoute.java

```diff
@@ -48,11 +48,13 @@
     private final int hashCode;
     private final int complexity;
 
+    private final boolean allowDeferredException;
```

Comment:
Would it be better choice if I use `isFallback` variable instead of this variable? 🤔

Reply:
Yeah, I think that makes sense if that's the case, but it'd be nice if we add some comment about why we use `isFallback` where `allowDeferredException` was used before.

Reply:
Thank you @trustin. Will add a comment :-)